public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -0,0 +1,638 @@
+  null, 
+  `app.component.ts,
+  app.component.html,
+  hero.ts,
+  hero-dashboard.component.ts,
+  hero-dashboard.component.html,
+  hero-detail.component.ts,
+  hero-status.pipe.ts`)
+
+:marked
+  At this point we don't seem much. We have a header, space for the HQ alerts,
+  lanes for each hero status, and a button.
+  
+figure.image-display
+  img(src='/resources/images/devguide/structural-directives/dashboard-empty.png' alt="empty dashboard")
+
+:marked
+  But first, let's dive a bit deeper into DOM.
+
+.l-main-section  
+:marked
+  ## A tale of three DOMs
+  
+  We are used to seeing only one DOM - the one containing the elements we see
+  rendered the browser. 
+  But we have now also seen that the DOM we declare when we use a component 
+  is superseded by the components own template DOM.
+  So what's the sense behind this?
+  
+  The answer lies in [Shadow Dom](http://webcomponents.org/polyfills/shadow-dom/),
+  a part of the [Web Components Spec](http://w3c.github.io/webcomponents/spec/shadow/).
+  Angular uses Shadow DOM emulation by default, but can be configured to either
+  use a native browser Shadow DOM implementation, or to not use it at all.
+  
+  Shadow DOM lets Angular 2 encapsulate a components DOM. This is why styles
+  we declare for a component don't leak out and affect other components.
+  Everything is nice and tidy inside our components.
+  
+  According to the spec, we have can have a total of *three* DOM trees inside an
+  element: **light DOM**, **shadow DOM** and **composed DOM**.
+  
+  **Light DOM** is what we see when *using* a component. It's what component
+  consumers interact with, and how they can give component creators some DOM
+  for them to use. In our previous example, this was the light DOM:
+  
++makeExample('structural-directives/ts/src/app/structural-directives.component.html', 'my-component-1')(format=".")
+
+:marked
+  **Shadow DOM** is what we see when *creating* a component. It's what component
+  creators interact with, and what they use to structure their components.
+  Templates is where we keep our shadow DOM in angular.
+
++makeExample('structural-directives/ts/src/app/my-component.component.ts')(format=".")
+
+:marked
+  **Composed DOM** is what the browser ultimately renders, and thus what we see
+  in the browser.
+  
+figure.image-display
+  img(src='/resources/images/devguide/structural-directives/my-component-content.png' alt="my-component content")
+
+:marked
+  So when we talked about transclusion and using externally provided DOM inside
+  our components, what we meant was that we were going to declare elements in 
+  the *light DOM* and use them from within our *shadow DOM*, creating the 
+  *composed DOM* that the browser will render. 
+  
+  Another way to think about it is that transcluded elements are being 
+  **projected** from the light DOM to the shadow DOM.
+  
+  The transcluded light DOM elements still remain in their original context as
```

Comment:
Again, I think we should avoid "transclusion" as a term entirely. This might confuse ppl, because most developers only know this word in the context of Angular 1 transclusion.